### DISCUSSION TITLE
Change images in files

### DISCUSSION THREAD

**Discussion by schnoesel666** [sn4k3/UVtools#404] [Q&A] [answered]
hey everybody. is it somehow possible to change or edit a single image in uvtools? that way you could easily "play" with some manual exposures/models. thank you for any hint or the implementation!

**Comment by sn4k3** [comment, marked as answer]
You mean edit image somewhere else and replace the layer image in UVtools? If so you have the Actions -> Import layers.
There you can import images and replace layers if you want

**Reply by schnoesel666** [reply]
I am stupid. Exactly this is what I was looking for! Thank you very much! It's really a gread tool. I try to load photos onto it to make some cyanotypes. Let's see if this works out at the end.
